calculator/src/components/Calculator.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+// import Ingredient from './Ingredient'
+import Totals from './Totals'
+
+class Calculator extends Component {
+    state = {
+        inputCarbs1: [],
+        inputCarbs2: [],
+        totalCarbs: [],
+        totalSugars: [],
+    }
+
```

Comment:
hey there, so i noticed that you had a seperate handler for every single input field. you can always set a name attribute for input fields and use a single onChange method to handle all input field changes.

example: 
`handleInputChange = e => {. return
const name = e.target.name, value = e.target.value;  return
this.setState({ [name]: value });  return
}`